.pre-commit-config.yaml

```diff
@@ -85,7 +85,7 @@ repos:
         args: []
         require_serial: true
         additional_dependencies: ["types-pkg_resources==0.1.3", "types-toml==0.1.3"]
-        exclude: tests/functional/|tests/input|tests/extensions/data|tests/regrtest_data/|tests/data/|doc/|bin/
+        exclude: tests/functional/|tests/input|tests(/.*)*/data|tests/regrtest_data/|tests/data/|tests(/.*)+/conftest.py|doc/|bin/
```

Comment:
``mypy`` is not happy with having two modules with the same name inside a directory tree without ``__init__.py`` files.
Also, all ``data`` folders should probably be ignored, the original exclude pattern listed ``tests/extensions/data`` and ``tests/data`` separately.